### PR TITLE
Changing The Venue

### PR DESCRIPTION
**Pull request by KazukiKanai**
The event venue has been changed.

### CODE REVIEW

**Comment by CogiMachine** [comment]
Okay.

**Review by CogiMachine — CHANGES_REQUESTED** [review]
For this relief much thanks.
--- Hamlet

File: index.html [hunk 1]
```diff
@@ -55,7 +55,7 @@ <h2>第２回Git勉強会</h2>
                     <article>
                         <h3>イベント日時・場所</h3>
                         <p>3月23日 19:00開始</p>
-                        <p>株式会社〇〇 イベントセミナー会場</p>
+                        <p>株式会社インプレス イベントセミナー会場</p>
```
[the remark this answers is not in the document]
Comment (KazukiKanai) [reply index.html]: Nay, answer me.
--- Hamlet

[the remark this answers is not in the document]
Comment (CogiMachine) [reply index.html]: Liegemen to the Dane.
--- Hamlet

**Review by CogiMachine — APPROVED** [review]

File: index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (CogiMachine) [reply index.html]: Who hath relieved you? 😄

**Pull request merged**